README.md

```diff
@@ -464,7 +464,57 @@ remote_state = {
 ``` 
 
 The result is that when you run `terragrunt` commands in the `qa/my-app` folder, you get the `lock` settings from the 
-parent, but the `remote_state` settings of the child. 
+parent, but the `remote_state` settings of the child.
+
+### Environment variables replacement
+
+With terraform is possible to set variables via Environment variables. This allows to control similar infrastructures with the same configuration files. Is also possible to use these variables in `.terragrunt` file by using the `get_env()` helper function.
+
+```hcl
+remote_state = {
+  backend = "s3"
+  config {
+    encrypt = "true"
+    bucket = "${get_env(ENVIRONMENT_VARIABLE, 'development')}-bucket"
```

Comment:
Shouldn't the environment variable name be quoted? I think that would be more idiomatic for HCL syntax.